mzLib/Proteomics/RetentionTimePrediction/Chronologer/ChronologerRetentionTimeEstimator.cs

```diff
@@ -0,0 +1,141 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using TorchSharp;
+
+namespace Proteomics.RetentionTimePrediction.Chronologer;
+public static class ChronologerRetentionTimeEstimator
+{
+    /// <summary>
+    /// Uses the Chronologer model to predict the retention time of a peptide (reported in % of ACN).
+    /// Only modifications present in the Chronologer dictionary are supported.
+    /// <code>
+    /// "Carbamidomethyl on C"
+    /// "Oxidation on M"
+    /// "Glu to PyroGlu"
+    /// "Phosphorylation on S"
+    /// "Phosphorylation on T"
+    /// "Phosphorylation on Y"
+    /// "Accetylation on K"
+    /// "Succinylation on K"
+    /// "Ubiquitination on K"
+    /// "Methylation on K"
+    /// "Dimethylation on K"
+    /// "Trimethylation on K"
+    /// "Methylation on R"
+    /// "Dimethylation on R"
+    /// </code>
+    /// </summary>
+    /// <param name="baseSequence"></param>
+    /// <param name="fullPeptideSequence"></param>
+    /// <returns></returns>
+    public static double PredictRetentionTime(string baseSequence, string fullPeptideSequence)
+    {
```

Comment:
Ambiguous PSMs will have a null full sequence until after PEP. PEP uses the retention time prediction. How will we handle this in MetaMorpheus?

Reply:
Ambiguous PSMs are resolved before getting to this point.